app/styles/app.scss

```diff
@@ -761,7 +761,6 @@ input[type="email"],
 input[type="month"],
 input[type="number"],
 input[type="password"],
-input[type="search"],
```

Comment:
I needed to remove this from the app's css because it was interfering with the search addon's styles.

I don't know if I have the CSS knowledge to get the addon's styles to "win" this battle so either we're ok to removing this line or I need some help 😖 

Things seem to work ok removing this line. 

/cc @MelSumner

Reply:
Right now the app's app.scss seems to be about 1500 LOC which probably means we copied this over from elsewhere. It might be better if we can expose a main stylesheet from the styleguide addon that we can import & then add our own page specific styles (which might not be a lot).

Reply:
yes @sivakumar-kailasam it was copied from the **output** of the guides snapshot.

if we're happy to continue with this as is (because it's working) we can do a CSS upgrade push after the initial release 👍

Reply:
If we're too close to the release, we can revisit this later.

Reply:
@sivakumar-kailasam I don't think it's quite as simple as being "too close to release" 🤔 the quest issue is done now and I think we can have it ready to swap over without putting a "feature freeze" on it, more improvements can land before we do the DNS swap over I mean.

Secondly, to be totally honest I don't understand the issue enough to know how important it is 😖 I would tend to defer to other people on the team to discuss/decide on this one (and yes that means deferring to you too 😂 )

Reply:
I brought this up only because each app ended up copying and customising styles over the last year in its own styles folder that it would take many/all of us to migrate common styles from the individual apps into the addon. 

In case you have the bandwidth to do this now please try to move them (If you don't, I understand ✌️ ).

Reply:
using the es-navbar component in ember-styleguide should resolve any of the search CSS issues, right?